rosbag2_storage/include/rosbag2_storage/storage_interfaces/base_info_interface.hpp

```diff
@@ -30,6 +32,13 @@ class ROSBAG2_STORAGE_PUBLIC BaseInfoInterface
   virtual ~BaseInfoInterface() = default;
 
   virtual BagMetadata get_metadata() = 0;
+
+  /**
+   * Retrieves the relative path to the backing of the storage plugin.
+   *
+   * \returns the relative path.
+   */
+  virtual std::string get_relative_path() const = 0;
```

Comment:
This is up for discussion, but when I think about it, I believe this function makes more sense on the `base_io_interface` then on the `info` one. 

We merged the other one already, but I think the `get_storage_identifier()` could actually be part of the info interface then.

Reply:
Opened issue #197 regarding this.